content/300-guides/300-upgrade-guides/800-upgrade-from-prisma-1/03-upgrading-the-prisma-layer.mdx

```diff
@@ -229,7 +229,7 @@ endpoint: http://localhost:4466/myproject/dev
 Based on these connection details, you need to configure the `DATABASE_URL` environment variable inside your `.env` file as follows:
 
 ```js file=.env
-DATABASE_URL="postgresql://janedoe:randompassword@localhost:5432/prisma?schema=myproject$dev"
+DATABASE_URL = 'postgresql://janedoe:randompassword@localhost:5432/prisma?schema=myproject$dev'
```

Comment:
Hm I think the `.env` file actually needs to be exactly in this format as it was to work. We would need to add an exclusion for this somehow.

Reply:
@janpio prettier formatted this block as `JS`, adding `{/* prettier-ignore */}` should fix that.

Reply:
Can we somehow configure it to ignore `js file=.env` completely?

Although, we probably should not say this is `JS` then in the first place and find a better formatter! This is actually a bug then.